src/compiler/crystal/command.cr

```diff
@@ -309,7 +307,10 @@ class Crystal::Command
       end
 
       unless no_codegen
-        opts.on("--emit [#{VALID_EMIT_VALUES.join('|')}]", "Comma separated list of types of output for the compiler to emit") do |emit_values|
+        valid_emit_values = Compiler::EmitTarget.names
+        valid_emit_values.map! { |v| v.tr("_", "-").downcase }
```

Comment:
You should use `Char`s instead of `String`s here.

Reply:
Yeah that's what I did originally, but there is only `tr(String, String)` not `tr(Char, Char)`

Reply:
There's `gsub(Char, Char)`